tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/RelationType/RelationType.spec.ts

```diff
@@ -1,55 +0,0 @@
-import {test} from '@umbraco/playwright-testhelpers';
```

Comment:
Why was this deleted?

Reply:
Currently, there are no endpoints for create/update/delete Relation Type, so I deleted this file

Reply:
I think we should uncomment the file instead of deleting it. 🤔